tests/func/test_add.py

```diff
@@ -272,6 +279,8 @@ def test_add_external_dir(tmp_dir, dvc, workspace, hash_name, hash_value):
     assert (tmp_dir / "dir.dvc").read_text() == (
         "outs:\n"
         f"- {hash_name}: {hash_value}\n"
+        "  size: 11\n"
+        "  nfiles: 2\n"
         "  path: remote://workspace/dir\n"
```

Comment:
This should really be:
```
- path: ...
  md5: ...
  size: ...
  nfiles: ...
```
added a checkbox to https://github.com/iterative/dvc/issues/4841 , but we could do this right away, as it will only affect the newly written files, which is a bit annoying but not that bad.